extensions.json

```diff
@@ -914,6 +914,8 @@
     "timeout": 10,
     "custom": [
       "[ $VERSION = '0.2.1' ] && npm install --save-dev webpack@5.70.0 webpack-cli@4.9.2 || true",
+      "sed -i '/\"extensionDependencies\":\\s*\\[/,/\\]/{/^\\s*\".*\",\\?$/d}' package.json || true",
+      "sed -i '/\"extensionDependencies\":\\s*\\[/a \"muhammad-sammy.csharp\"' package.json || true",
```

Comment:
@manuth can we achieve this by using `jq` or something similar? Would be nice to make this custom code more readable. Otherwise this LGTM.

Reply:
I have tried this, but it has some escaping errors.

```bash
cat package.json | jq --arg deps ' [ \"muhammad-sammy.csharp\" ] ' '.extensionDependencies = $deps'
```

Reply:
Awesome! I didn't know jq was installed by default.
I'll add this asap!